### ISSUE TITLE
SimpleButton - spinner not showing

### ISSUE THREAD

**Issue by aford22** [stsrki/Blazorise#103]
Hello,

I am using the following code:
```
<SimpleButton Color="Color.Primary" IsLoading="true">Login</SimpleButton>
```

The button shows on the page but the spinner is not showing at all. Am I missing something?

**Comment by stsrki** [comment]
@aford22 I don't see the code. Can you please re-post it?

**Comment by stsrki** [comment]
What CSS framework yo're using?

**Comment by aford22** [comment]
Material. Please see the relevant sections in my _Host.cshtml, which may well be incorrect!

```
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" />
    <link href="~/css/blazorise.css" rel="stylesheet" />
    <link href="~/css/blazorise.bootstrap.css" rel="stylesheet" />
    <link href="~/css/blazorise.material.css" rel="stylesheet" />
    <link href="~/css/blazorise.sidebar.css" rel="stylesheet" />
    <link href="~/css/blazorise.snackbar.css" rel="stylesheet" />
    <link href="~/css/demo.css" rel="stylesheet" />
    <link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,500,500i,700,700i|Roboto+Mono:300,400,700|Roboto+Slab:300,400,700" rel="stylesheet">
    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
    <link href="~/css/material.min.css" rel="stylesheet">
    <link href="~/css/site.css" rel="stylesheet" />

    <script src="~/js/blazorise.js"></script>
    <script src="~/js/blazorise.bootstrap.js"></script>
    <script src="~/js/blazorise.material.js"></script>
    <script src="~/js/blazorise.sidebar.js"></script>
    <script src="~/js/demo.js"></script>
```
Does this look right?

**Comment by stsrki** [comment]
It seems material from [daemonite](http://daemonite.github.io/material/) is still using an older version of Bootstrap and that version doesn't have spinner buttons implemented.
I think I could create something in the next version of Blazorise 0.7.6.

In the meantime maybe you could play a little with css and add some classes based on bootstrap [spinners documentation](https://getbootstrap.com/docs/4.3/components/spinners/#about)

**Comment by aford22** [comment]
Aha yes I see now that the css doesn't include it. If you could add in 0.7.6 that would be great!!!

Thanks for the quick response.

**Comment by stsrki** [comment]
Fixed in the version 0.7.6
